apps/src/lab2/levelEditors/exemplar/ExemplarSettings.tsx

```diff
@@ -15,6 +17,41 @@ interface ExemplarSettingsProps {
   appName: AppName;
 }
 
+const validationModeOptions: Partial<{
+  [key in AppName]: {label: string; value: string}[];
+}> = {
+  music: MUSIC_VALIDATION_MODE_OPTIONS,
+};
+
+const exemplarValidationDescriptions: Partial<
+  Record<AppName, React.ReactNode>
+> = {
+  music: (
```

Comment:
I would love to move this to a more Music specific constants type file, but wasn't sure where might be the appropriate place.

Reply:
Good question.  It feels like we should have a common location for the declaration of the dropdown labels (`apps/src/music/constants.ts` in this PR) and these additional descriptions, since they are all levelbuilder-only strings.  If we want `apps/src/lab2` to be truly lab-agnostic, then maybe these descriptions belong over in `apps/src/music` too, perhaps in a new `apps/src/music/levelEditors/` or similar?  That way, `apps/src/lab2` remains lab agnostic, while individual labs can provide additional levelbuilder content and functionality. 

Separately, wondering whether `ReacNode` is a little implementation-specific for exported constants, and whether we could consider something like markdown for richer descriptions if necessary.